generators/chaincode/templates/javascript/index.js

```diff
@@ -17,4 +17,6 @@
 
 'use strict';
 
-module.exports.Chaincode = require('./lib/chaincode');
+const ContractA = require('./lib/contract-a.js')
```

Comment:
The class isn't called ContractA? It's called MyContract, which I prefer. The file names and imports should reflect that.